config/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM consul:1.7.1
+
```

Comment:
WDYT about structure your infra configurations and Dockerfiles, for example:
```
--infra
    -- consul
        agent-config.json
        Dockerfile
    -- ...
```

Reply:
moved to infra/consul/